index.js

```diff
@@ -1,61 +1,95 @@
-const addBook = document.getElementById("add-book");
+/*library array to store Book objects */
+const library = [];
+
+const addBookBtn = document.getElementById("add-book");
 const bookContainer = document.getElementsByClassName("book-container");
 const inputBook = document.getElementsByClassName("input-book");
-const readBook = document.getElementById("read-book");
 
-addBook.addEventListener("click", function () {
+class Book {
+  constructor(title, author, pages, haveRead) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.haveRead = haveRead;
+  }
+}
+
+/** eventListner to add book into bookContainer */
+addBookBtn.addEventListener("click", function () {
+  if (
+    inputBook[0].value === "" ||
+    inputBook[1].value === "" ||
+    inputBook[2].value === ""
+  ) {
+    alert("please Enter all The Details.");
+  } else {
+    library.push(
+      new Book(
+        inputBook[0].value,
+        inputBook[1].value,
+        inputBook[2].value,
+        (haveRead = true)
```

Comment:
I need a suggestion on changing haveRead = true 
purpose:
when user gives input if he he clicks on "read" button haveRead should be true, If he doesn't click on read button haveRead should be false, so that it will reflect on book in bookContainer.
I have tried using addEventListner but, it is not working